torchtune/models/llama3_2_vision/_component_builders.py

```diff
@@ -679,7 +675,6 @@ def lora_llama3_2_vision_projection_head(
     num_hidden_inputs: int,
     # LoRA args
     apply_lora_to_mlp: bool,
-    apply_lora_to_output: bool,
```

Comment:
I think this should stay in the projection head, it just doesn't make sense in the CLIP model